configure

```diff
@@ -16806,7 +16811,7 @@ do
 done
 
 
-SRCDIRS="Parser Parser/pegen Objects Python Modules Modules/_io Programs"
+SRCDIRS="Parser Objects Python Modules Modules/_io Programs"
```

Comment:
Is this change on purpose or bad merge?

Reply:
A previous commit from another project left `configure` out of sync with ``configure.ac``.  While this corrects that, I'm pushing a separate PR (#21152 )  to fix it right away.